packages/xod-client-browser/src/containers/App.jsx

```diff
@@ -249,6 +249,8 @@ class App extends client.App {
         onClick(items.undo, this.props.actions.undoCurrentPatch),
         onClick(items.redo, this.props.actions.redoCurrentPatch),
         items.separator,
+        onClick(items.selectall, this.props.actions.selectAll),
+        items.separator,
```

Comment:
This makes menu overflown. Also, a non-relevant in Linux command+A is shown

![screenshot from 2018-10-25 12-31-16](https://user-images.githubusercontent.com/146383/47490802-2ab19880-d852-11e8-81e2-6dc59614fc7c.png)